### PR TITLE
[4.0] Irrelevent portions of code removed

### PR DESCRIPTION
**Pull request by saumyasarkar11**
Pull Request for Issue #32725 .

### Summary of Changes

1. && $this->item->pagination removed from https://github.com/joomla/joomla-cms/blob/4.0.0-beta7/components/com_content/tmpl/article/default.php#L43.

2. Brackets around OR removed from https://github.com/joomla/joomla-cms/blob/4.0.0-beta7/components/com_content/tmpl/article/default.php#L49-L51.

### Documentation Changes Required

None.

### CODE REVIEW

File: components/com_content/tmpl/article/default.php [hunk 1]
```diff
@@ -40,15 +40,15 @@
...
 	</div>
 	<?php endif;
-	if (!empty($this->item->pagination) && $this->item->pagination && !$this->item->paginationposition && $this->item->paginationrelative)
+	if (!empty($this->item->pagination) && !$this->item->paginationposition && $this->item->paginationrelative)
 	{
 		echo $this->item->pagination;
 	}
 	?>
 
 	<?php // Todo Not that elegant would be nice to group the params ?>
```
Comment (ReLater) [inline components/com_content/tmpl/article/default.php]: Please remove this comment, too.
```suggestion

```

Comment (saumyasarkar11) [reply components/com_content/tmpl/article/default.php]: I had removed it initially but @joomdonation asked me to keep it.

Comment (ReLater) [reply components/com_content/tmpl/article/default.php]: Please remove it. In issue https://github.com/joomla/joomla-cms/issues/32725#issuecomment-801887572 you can see: **_can be removed_** Thank you.

Comment (saumyasarkar11) [reply components/com_content/tmpl/article/default.php]: ok

**Comment by saumyasarkar11** [comment]
So @ReLater according to you, this is the correct one?

**Comment by ReLater** [comment]
I have tested this item :white_check_mark: successfully on e0f22cfe169f413f5bf146e11e5cacb9959b172f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32746">issues.joomla.org/tracker/joomla-cms/32746</a>.</sub>

**Comment by saumyasarkar11** [comment]
> I have tested this item  successfully on [e0f22cf](https://github.com/joomla/joomla-cms/commit/e0f22cfe169f413f5bf146e11e5cacb9959b172f)
> 
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/32746](https://issues.joomla.org/tracker/joomla-cms/32746).

Is it RTC now?

**Comment by ReLater** [comment]
> Is it RTC now?

PRs need 2 successful tests before they go RTC. Always tests of others.

Thank you! Small changes with great effort ;-)

**Comment by saumyasarkar11** [comment]
> > Is it RTC now?
> 
> PRs need 2 successful tests before they go RTC. Always tests of others.
> 
> Thank you! Small changes with great effort ;-)

Welcome!

**Comment by joomdonation** [comment]
I wonder why we don't make the same change (remove **&& $this->item->pagination**) on the other lines on same file ?

https://github.com/joomla/joomla-cms/blob/4.0-dev/components/com_content/tmpl/article/default.php#L117
https://github.com/joomla/joomla-cms/blob/4.0-dev/components/com_content/tmpl/article/default.php#L139

**Comment by ReLater** [comment]
> I wonder why we don't make the same change (remove && $this->item->pagination) on the other lines on same file ?

Agree. I forgot to mention that. Just noted in in my todo...

**Comment by alikon** [comment]
@saumyasarkar11  can you please do the requested changes  https://github.com/joomla/joomla-cms/pull/32746#issuecomment-802789442 too

**Review by saumyasarkar11 — COMMENTED** [review]
Requested changes executed! Please review @ReLater @joomdonation @alikon

**Comment by richard67** [comment]
Some testing instructions would be required. We have the corresponding head lines in the template for our pull requests not without a reason.

**Comment by ReLater** [comment]
I have tested this item :white_check_mark: successfully on 881f4612465f567fb7430f0425f0dfa5d37d07c3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32746">issues.joomla.org/tracker/joomla-cms/32746</a>.</sub>

**Comment by ReLater** [comment]
> Some testing instructions would be required.

Code review should be sufficient. See also description in issue https://github.com/joomla/joomla-cms/issues/32725#issue-834271832

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on 881f4612465f567fb7430f0425f0dfa5d37d07c3<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32746">issues.joomla.org/tracker/joomla-cms/32746</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32746">issues.joomla.org/tracker/joomla-cms/32746</a>.</sub>

**Comment by drmenzelit** [comment]
Thanks @saumyasarkar11 and congratulations to your first merged PR in Joomla

**Pull request merged**